### PR TITLE
Added ransom solution

### PR DESCRIPTION
**Pull request by prestonvanloon**
I didn't see this folder in here, but I took the time to type this one out so I figured I would submit!

### CODE REVIEW

**Comment by bcjordan** [comment]
Awesome, looks great!

File: ransom/RansomTest.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class RansomTest {
+
+    @Test
+    public void testCanRansom() throws Exception {
+        boolean yayRansom = Ransom.canRansom("dying wool is what you will be doing", "you will be dying");
+        System.out.println("You " + (yayRansom ? "can" : "can't") + " write a ransom letter");
+        boolean failRansom = Ransom.canRansom("can you believe Justin Beiber's new shirt?", "give me all your money");
```
Comment (bcjordan) [inline ransom/RansomTest.java:10]: :laughing:

**Comment by dideler** [comment]
Missing README w/ problem description :cry:

**Comment by prestonvanloon** [comment]
Oh yeah, this is from lecture 48. The problem is:

Given a magazine and a ransom note, figure out if you can cut some words out of the magazine to write the ransom note.

**Pull request merged**